lib/rules/helpers.js

```diff
@@ -0,0 +1,27 @@
+const tidioModules = [
```

Comment:
maybe we should pass this as param to rule config (in case new global module is introduced)?

good enough for now, your call

Reply:
options added